source/operations/concepts.rst

```diff
@@ -8,126 +8,173 @@ Core Operational Concepts
-   impact despite the loss of up to half (``N/2``) of the total drives in the
-   deployment.
+What are the components of a MinIO Deployment?
+----------------------------------------------
+
+A MinIO deployment consists of a set of storage and compute resources running one or more :mc:`minio server` nodes that together act as a single object storage repository. 
+
+A standalone cluster consists of a single Server Pool with a single :mc:`minio server` node. 
+Standalone clusters are best suited for initial development and evaluation. 
+
+A MinIO deloyment can run directly on a physical device in a ``baremetal`` scenario.
+Or, MinIO might run within a virtual machine on a cloud service, such as using Docker, Podman, or Kubernetes.
+MinIO can run locally, on a private cloud, or in any of the many public clouds available on the market.
+
+The specific way you design, architect, and build your system is called the system's ``topology``.
+
+What system topologies does MinIO support?
+------------------------------------------
+
+MinIO can deploy to three types of topologies:
+
+#. :ref:`Single Node Single Drive <minio-snsd>`, one MinIO server with a single drive or folder for data
+
+   For example, testing on a local PC using a folder on the computer's hard drive. 
+#. :ref:`Single Node Multi Drive <minio-snmd>`, one MinIO server with multiple mounted drives or folders for data
+
+   For example, a single container with two or more mounted volumes.
+#. :ref:`Multi Node Multi Drive <minio-mnmd>`, multiple MinIO servers with multiple mounted drives or volumes for data
+
+   For example, a production deployment using Kubernetes to manage and deploy pods with multiple persistent volume claims. 
+
+
+How does a distributed MinIO deployment work?
+---------------------------------------------
+
+A distributed deployment makes use of the resources of more than one physical or virtual machine's compute and storage resources.
+In modern situations, this often means running MinIO in a private or public cloud environment, such as with Amazon Web Services, the Google Cloud Platform, Microsoft's Azure platform, or many others.
 
 .. _minio-intro-server-pool:
 
-:ref:`Server Pool <minio-intro-server-pool>`
-   A set of MinIO :mc:`minio server` nodes which pool their drives and
-   resources for supporting object storage/retrieval requests. Server pools
-   support horizontal expansion for MinIO deployments.
+How does MinIO manage multiple virtual or physical servers?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While testing MinIO may only involve a single drive on a single computer, most production MinIO deployments use multiple compute and storage devices to create a high availability environment.
+A server pool a set of :mc:`minio server` nodes that pool their drives and resources to support object storage write and retrieval requests.
+
+MinIO supports adding one or more server pools to existing MinIO deployments for horizontal expansion.
```

Comment:
One key thing is each server pool is effectively a fault domain. 

If I hvae 3 server pools and one goes offline, objects stored in the other two remain accessible.

Reply:
As long as people don't take that to mean that all objects remain available, since objects are written to erasure sets within a specific pool.

Reply:
Added a line about objects going to the same server pool.